data/database-schema.php

```diff
@@ -153,6 +153,13 @@ class SCHEMA
         'EmailListID' => 'INT UNSIGNED NOT NULL',
         'EditList' => 'BOOLEAN NOT NULL',
         'ChangeAccess' => 'BOOLEAN NOT NULL',
+    ],
+
+    'ConComPermissions' => [
```

Comment:
You'll want to add an index on `Permission` -- we'll be searching on it a lot, I suspect.

Reply:
Ok change made to support that and pushed.